src/ShopBundle/objects/TCMSTableEditor/TCMSShopTableEditor_ShopArticle.class.php

```diff
@@ -176,6 +183,9 @@ protected function isNewVariant()
     /**
      * we overwrite the insert method, so that when inserting variants, we realy perform
      * a copy of the parent.
+     *
+     * @psalm-suppress InvalidReturnType
+     * @FIXME This method should return TCMSstdClass with the object data similar to it's parent, not `void`.
```

Comment:
Return value of this method differs from its parent - we should probably return a `TCMSstdClass` here to